csharp-intermediate-topics/DifferencesBetweenDynamicTypes/DifferencesBetweenDynamicTypes/Person.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Dynamic;
+
+namespace DifferencesBetweenDynamicTypes;
+
+public class Person : DynamicObject
+{
+    private Dictionary<string, object> personalInformation;
```

Comment:
This doesn't respect our naming convention, it should start with underscore, and it can be readonly. 
The dictionary should accept null values as well. 

```suggestion
    private readonly Dictionary<string, object?> _personalInformation;

```